js/stores/windowStore.js

```diff
@@ -664,6 +664,8 @@ const doAction = (action) => {
           if (action.notify) {
             ipc.send('autofill-popup-hidden', action.tabId)
           }
+        } else {
+          windowState = windowState.delete('contextMenuDetail')
```

Comment:
@darkdh OK fixed- because I am not sure if the order matters, I moved the delete to above this check